### ISSUE TITLE
Baseline Experiment - Viewpoint on ShapeNet (Linear Probes)

### ISSUE THREAD

**Issue by druhinb** [druhinb/LatentInvestigation#5]
Develop the training and evaluation loop for the linear probing setup on ShapeNet for viewpoint prediction. The models are already pretrained, this step is more of a "fine-tuning" step.

**Subtasks** 

- Training script using AdamW, MSE loss.
- Evaluation script calculating Mean Absolute Error (MAE) for azimuth/elevation.
- Systematic hyperparameter tuning (LR, weight decay, batch size) using a validation set.
